linux/src/timer_linux.c

```diff
@@ -180,6 +180,28 @@ double timer_get_elapsed_ms(TIMER_HANDLE timer)
     return result;
 }
 
+double timer_global_get_elapsed_s(void)
+{
+    double result;
+    struct timespec elapsed_time;
+
+    /* Codes_SRS_TIMER_27_001: [ timer_global_get_elapsed_s shall return the elapsed time in seconds from a start time in the past. ]*/
```

Comment:
Should this be `SRS_TIMER_LINUX_27_001`? Also, `SRS_TIMER_LINUX_27_003` seems to be missing.